app/src/main/res/drawable/yellow_bar.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android" android:shape="rectangle">
+
+    <solid android:color="@color/colorApp"/>
+    <corners android:radius="@dimen/size_10"/>
+    <size android:width="@dimen/size_5"
+        android:height="25dp"/>
```

Comment:
you're already using the dimens file for the width, why not using it for height?